### DISCUSSION TITLE
Scan Barcode

### DISCUSSION THREAD

**Discussion by lookme2** [inventree/InvenTree#3126] [Q&A] [answered]
I'm not sure this is a bug or not.  So I thought I would start here.  After updating to 7.0 and trying to scan a barcode I get "Server error"

The question is do I have something setup wrong?  I seem to remember something about this moving to a plug in but cant seem to find that in the docs

Frankie

**Comment by SchrodingersGat** [comment, marked as answer]
- Are you saying you updated the server to `0.7.0`, or the app?
- The app reports "server error"?

**Reply by lookme2** [reply]
@SchrodingersGat yes I have updated to 0.7.0 and when I scan via the web page I get "Server error" back.  The scan does not work.

**Reply by matmair** [reply]
@lookme2 do you still experience the issue in 0.8.x?

**Reply by lookme2** [reply]
@matmair  sorry i was out of the office all day yesterday.  I am now on 0.8.2 and the scanning part seems to work fine for me.
